src/event_loop.rs

```diff
@@ -66,6 +67,8 @@ impl EventLoopBuilder<()> {
     }
 }
 
+static EVENT_LOOP_CREATED: OnceCell<Mutex<bool>> = OnceCell::new();
```

Comment:
Shouldn't we refactor that to `AtomicBool` or simply `Mutex` given that it could be changed now? The `Mutex` is const anyway now.